src/ProjectConfiguration.test.ts

```diff
@@ -402,7 +403,7 @@ it('validates the default language is a valid language tag', () => {
     expect.objectContaining({
       category: DiagnosticCategory.Error,
       messageText:
-        'Default language is an invalid language tag: _really_not_bcp_47_',
+        'Default language is an invalid language tag: _really_not_bcp_47_. Must be de-DE, en-US, es-ES, fr-FR, it-IT, ja-JP, ko-KR, nl-NL, sv-SE, zh-CN or zh-TW.',
```

Comment:
should this be iterated from languageTag.ts => supportedTags ?

Reply:
In general until now we've typically avoided using constants in tests like that so that it's clear when we change them what tests it affects. I'm kinda on the fence, I see the utility in using the constant, but it is nice to see what the user-facing impacts were too.

Reply:
as long as we don't see a painful fix that we always have to make, i'm fine with leaving it as-is.